<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RSC-600M-3C固定床和反应釜系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/ui_ctrls/Res/icon_device.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QAxWidget" name="widget_workflow">
    <property name="enabled" stdset="0">
     <bool>false</bool>
    </property>
    <property name="geometry" stdset="0">
     <rect>
      <x>61</x>
      <y>0</y>
      <width>1200</width>
      <height>1100</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_control" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>0</y>
      <width>1200</width>
      <height>1100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>900</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>600</width>
       <height>600</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>60</width>
      <height>1100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#333333;color:#FFFFFF</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::IconMode</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <property name="uniformItemSizes">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_trend" native="true">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>0</y>
      <width>1200</width>
      <height>1100</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#FFFFFF</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="widget_distributor">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>0</y>
      <width>1200</width>
      <height>1100</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_history" native="true">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>0</y>
      <width>1200</width>
      <height>1100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#FFFFFF</string>
    </property>
   </widget>
   <zorder>widget_workflow</zorder>
   <zorder>listView</zorder>
   <zorder>widget_trend</zorder>
   <zorder>widget_control</zorder>
   <zorder>widget_distributor</zorder>
   <zorder>widget_history</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
